src/quat.js

```diff
@@ -447,32 +447,75 @@ export function fromMat3(out, m) {
 }
 
 /**
- * Creates a quaternion from the given euler angle x, y, z.
+ * Creates a quaternion from the given euler angle x, y, z using the provided extrinsic order for the conversion.
  *
  * @param {quat} out the receiving quaternion
  * @param {x} Angle to rotate around X axis in degrees.
- * @param {y} Angle to rotate around Y axis in degrees.
  * @param {z} Angle to rotate around Z axis in degrees.
+ * @param {y} Angle to rotate around Y axis in degrees.
+ * @param {order} Intrinsic order for conversion, so far only 'xyz' and 'zyx' is supported.
```

Comment:
Looks like some of these comments are out of date now. The description says extrinsic order but the argument says intrinsic with only two options available and y is now listed as the third argument.

Reply:
Oops, thanks for catching that. Fixed.